elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -224,6 +224,9 @@ protected PersistentResource(T obj, PersistentResource<?> parent, RequestScope r
         this(obj, parent, requestScope.getUUIDFor(obj), requestScope);
     }
 
+    protected boolean isNewlyCreated() {
```

Comment:
A function level documentation is appreciated since it might be called by another class within the same package.  Also unluckily another [variable](https://github.com/yahoo/elide/blob/master/elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java#L1459) in the same file has declared the name `isNewlyCreated`, you might consider changing either of their name to avoid confusion.

Reply:
Actually, I think this function should be used in place of the variable that @tarrantzhang is referring to.